src/lib/core/components/visualizations/implementations/BarplotVisualization.tsx

```diff
@@ -649,16 +657,42 @@ function BarplotViz(props: VisualizationProps<Options>) {
       {/* Y-axis range control */}
       <div style={{ display: 'flex', flexDirection: 'row' }}>
         <div style={{ display: 'flex', flexDirection: 'column' }}>
-          <LabelledGroup label="Y-axis controls">
-            <div style={{ display: 'flex' }}>
-              <Toggle
-                label="Log scale:"
-                value={vizConfig.dependentAxisLogScale}
-                onChange={onDependentAxisLogScaleChange}
-                themeRole="primary"
-              />
+          {/* set Undo icon and its behavior */}
+          <div
+            style={{
+              display: 'flex',
+              flexDirection: 'row',
+              alignItems: 'center',
+            }}
+          >
+            <LabelledGroup label="Y-axis controls"> </LabelledGroup>
+            <div style={{ marginLeft: '-2.3em' }}>
+              <Tooltip title={'Reset to defaults'}>
+                <button
+                  onClick={handleDependentAxisSettingsReset}
+                  style={{
+                    width: undoSize,
+                    height: undoSize,
+                    background: 'none',
+                    border: 'none',
+                    padding: 0,
```

Comment:
I played around with an extra `margin-left: 2px` in dev tools and I think it looks a bit less crowded.

Reply:
Thank you for your tests, @bobular 👍 I will consider it